### PR TITLE
Final quiz page

### PR DESCRIPTION
**Pull request by athar05**
This PR contains:

- A result page which displays all the correct and wrong answers 

Do not check for mobile responsiveness

### CODE REVIEW

**Review by rahul31agrawal — APPROVED** [review]
Hi Aakhib

1. The collections of category in your qiz app is really good
2. I liked your final page which displays the user answer & correct answer
3. I suggest to remove the codes ("like &309 ") comming in some of the options
4. App works amazingly smooth.
Thanks

File: src/App.css [hunk 1]
```diff
@@ -272,6 +284,24 @@ a {
...
 
+/* result page begins */
+.result-body {
+  border: 5px solid var(--primary);
+  padding: 0.5rem 0.5rem;
+}
+
+.result-question-body{
+  margin: 0.5rem 0.5rem;
+  /* border: 1px solid var(--primary); */
```
Comment (HarshnaNHaswani) [inline src/App.css]: remove commented code if not required

File: src/pages/Result.js [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+      <div className="nav">
+        <Navbar />
+      </div>
+      <div className="body-container-result">
+        <ResultBody />
+      </div>
+      <div className="footer">
+        <Footer />
+      </div>
+    </div>
```
Comment (HarshnaNHaswani) [inline src/pages/Result.js:18]: instead of repeating common elements like nav and footer, you can put them in  a layout

**Review by HarshnaNHaswani — COMMENTED** [review]
- Clean code and folder structure
- Smooth, Intuitive UI
- Can disable option after the answer is selected on quiz page
- Do you really need separate folder for each component? It would have made sense if the folder contained logic, UI, and styling separately for each component. Here component has only 1 file
- can use ems in media queries and custom hooks for useContext
Overall, nicely done

File: src/App.css (hunk 1 above)
Comment (athar05) [reply src/App.css]: Sure, thanks

File: src/pages/Result.js (hunk 2 above)
Comment (athar05) [reply src/pages/Result.js:18]: I will look into it.

**Pull request merged**